docs/tutorials/configure/pet-photographer.md

```diff
@@ -0,0 +1,730 @@
+If your robot is connected, you will see a selection of available cameras.
+Select the camera you want to use, then click **Save config**
+
+![An instance of the webcam component named 'cam'](/tutorials/pet-photographer/webcam-component.png)
+
+### Add colorfilter component
+
+1. Click the **Components** subtab and click **Create component**.
+
+1. Then, select the `local modular resource` type from the list.
+   {{<imgproc src="/extend/modular-resources/configure/add-local-module-list.png" resize="300x" declaredimensions=true alt="The add a component modal showing the list of components to add with 'local modular resource' shown at the bottom">}}
+
+1. On the next screen:
+
+   1. Select the camera from the drop down menu.
+   1. Enter the {{< glossary_tooltip term_id="model-namespace-triplet" text="model namespace triplet">}} of your modular resource's [model](/extend/modular-resources/key-concepts/#models), `example:camera:colorfilter`.
+   1. Enter a name for this instance of your modular resource.
+      This name must be different from the module name.
+
+   {{<imgproc src="/tutorials/pet-photographer/add-colorfilter-module-create.png" resize="400x" declaredimensions=true alt="The add a component model showing the create a module step for a local color filter module">}}
+
+1. Click **Create** to create the modular resource component.
+
+1. Copy the following JSON configuration into the **Attributes** section:
+
+```json {class="line-numbers linkable-line-numbers"}
+{
+  "vision_service": "my_color_detector",
+  "actual_cam": "cam"
+}
+```
+
+![A component panel for a color filter modular resource with the attributes filled out for vision service and actual_cam](/tutorials/pet-photographer/colorfiltercam-component-attributes.png)
+
+### Configure data capture
+
+To add data capture for the colorfilter camera, click **Add Method** in the **Data Capture configuration** section of your color filter camera component.
+Toggle the **Type** dropdown menu, select **ReadImage**, and set the **Frequency** of the capture to `0.1`.
+Then, click **Save config**.
+
+![A component panel for a color filter modular resource with the attributes filled out for vision service and actual_cam as well as the data capture configuration capture set capture ReadImage at 0.1 frequency](/tutorials/pet-photographer/colorfiltercam-component.png)
+
+## Test your color filter camera
+
+To test that your color filter camera is capturing and filtering images properly, navigate to the **Control** tab on your robot's page.
+
+On the **colorfiltercam**'s panel, toggle **view colorfiltercam** to view your camera's live feed.
+Test the filter by moving a blue colored item within the camera's field of view.
+Then, go to the **Data** tab to view pictures that contain the blue colored item.
+
+![Filtered data tab contents from colorfiltercam showing only photos of dog with blue collar](/tutorials/pet-photographer/data-capture.png)
```

Comment:
So cute! Very small nit - but can we move this pic to the Next steps section since that's the part where we talk about the colored object on the pet?